PWGGA/GammaConv/AliAnalysisTaskGammaConvCalo.cxx

```diff
@@ -5553,12 +5561,17 @@ void AliAnalysisTaskGammaConvCalo::ProcessTrueMesonCandidates(AliAODConversionMo
         isTrueEta=kTRUE;
       }
     }
+    Double_t weightMatBudgetGamma = 1.;
+    if (fDoMaterialBudgetWeightingOfGammasForTrueMesons && ((AliConversionPhotonCuts*)fCutArray->At(fiCut))->GetMaterialBudgetWeightsInitialized()) {
+        weightMatBudgetGamma = ((AliConversionPhotonCuts*)fCutArray->At(fiCut))->GetMaterialBudgetCorrectingWeightForTrueGamma(TrueGammaCandidate0);
+    }
+
+    //For MC we fill now the historgram here with weights (true and reconstructed efficiency will be consistent)
+    fHistoMotherInvMassPt[fiCut]->Fill(Pi0Candidate->M(),Pi0Candidate->Pt(),fWeightJetJetMC*weightMatBudgetGamma);
```

Comment:
This histogram is filled twice when running JetAnalysis, isn't it? If that is true, removing L5206 + L5210-5212 should be enough I'd say? Could you please check